### DISCUSSION TITLE
GitHub Copilot not working tho my application has been accepted two weeks ago

### DISCUSSION THREAD

**Discussion by Brndis** [community/community#127115] [GitHub Education] [answered]
### Hi everyone,

I applied for the GitHub student program and happily got approved two weeks ago. Unfortunately I still cannot access GitHub Copilot with my account. Anyone knows how to get it working? 
Thanks in advance!

**Comment by Brndis** [comment, marked as answer]
I just had to activate it myself lol

link: https://github.com/github-copilot/signup/

**Reply by aidwiz** [reply]
yeah, true for me too. thanks for the answer. approval means you are approved but you need to activate the pro version manually (strange)
